src/org/etools/j1939_84/model/SpnGroup.java

```diff
@@ -23,6 +24,15 @@ public boolean isSatisfied(Collection<Integer> supportedSpnValues) {
         return supportedSpnValues.stream().anyMatch(spns::contains);
     }
 
+    public boolean containsMultiple(Collection<Integer> supportedSpnValues) {
+        Collection<Integer> matchingSpns = new ArrayList<>();
```

Comment:
this variable isn't used.

Reply:
It is used in the part 1 step 4 that is also up for review.